stdlib/public/Concurrency/AsyncStreamBuffer.swift

```diff
@@ -55,6 +55,7 @@ func _unlock(_ ptr: UnsafeRawPointer)
 @available(SwiftStdlib 5.1, *)
 extension AsyncStream {
   @safe
+  @usableFromInline
```

Comment:
tangential question regarding this change – i've for some time wondered if the internal storage for these types could in some way be unified into something like `_Storage<Element, E: Error>` since they have so much duplication. does this change make that idea less feasible?

Reply:
Maybe, for now we just need to land this hashable :)